avocado_vt/options.py

```diff
@@ -34,6 +34,9 @@
 from virttest.standalone_test import SUPPORTED_TEST_TYPES
 
 
+LOG_OPTS = logging.getLogger('avocado.vt.options')
```

Comment:
I think we can use the name `LOG` instead because it is the default logger for this module.